march_joint_inertia_controller/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(march_joint_inertia_controller)
+
+# add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
+  pluginlib
+  roscpp
+  std_msgs
+  message_generation
+)
+
+# find_package(Boost REQUIRED COMPONENTS system)
+
+# catkin_python_setup()
+
+# add_message_files(
+#   FILES
+#   Message1.msg
+#   Message2.msg
+# )
+
+# add_service_files(
+#   FILES
+#   Service1.srv
+#   Service2.srv
+# )
+
+# add_action_files(
+#   FILES
+#   Action1.action
+#   Action2.action
+# )
+
+# generate_messages(
+#   DEPENDENCIES
+#   std_msgs  # Or other packages containing msgs
+# )
+
+# generate_dynamic_reconfigure_options(
+#   cfg/DynReconf1.cfg
+#   cfg/DynReconf2.cfg
+# )
+
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES march_joint_inertia_controller
+#  CATKIN_DEPENDS controller_interface pluginlib
+#  DEPENDS system_lib
+)
+
+include_directories(
+# include
+  ${catkin_INCLUDE_DIRS}
+)
+
+# add_library(${PROJECT_NAME}
+#   src/${PROJECT_NAME}/march_joint_inertia_controller.cpp
+# )
+
+# add_dependencies(${PROJECT_NAME} ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+
+# add_executable(${PROJECT_NAME}_node src/march_joint_inertia_controller_node.cpp)
+
+# set_target_properties(${PROJECT_NAME}_node PROPERTIES OUTPUT_NAME node PREFIX "")
+
+# add_dependencies(${PROJECT_NAME}_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+
+add_library(joint_inertia_controller_lib src/joint_inertia_controller.cpp)
+target_link_libraries(joint_inertia_controller_lib ${catkin_LIBRARIES})
+
```

Comment:
You should remove all the comments and add this for installing the plugin properly:

```cmake
install(DIRECTORY include/${PROJECT_NAME}/
    DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
)

install(TARGETS ${PROJECT_NAME}
    ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
    LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
)

install(FILES controller_plugins.xml
    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
```